app/src/main/java/com/spoony/spoony/data/dto/response/GetPostResponseDTO.kt

```diff
@@ -0,0 +1,52 @@
+package com.spoony.spoony.data.dto.response
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class GetPostResponseDTO(
```

Comment:
```suggestion
data class GetPostResponseDto(
```
P1: 여기도!!! 노션 정독하고 와라~~~~

Reply:
수정했습니다~